DirectEmployers/MyJobs
# [PD-3105] map help text shows zero

## What's this PR do?
Map Text for analytics fixed for showing the amount of job views

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the analytics application

## How should this be manually tested?
Pull down the repository. Go into the analytics app and check the World and USA State Maps and hover over a specific Country or State and you will see the job views displaying the actual views instead of 0

## Any background context you want to provide?
Quick Fix Needed THIS WEEK
